hack/testing/entrypoint.sh

```diff
@@ -24,6 +24,7 @@
 
 source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
 source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+${EXTRA_ENV:-}
```

Comment:
?

Reply:
Most likely, a leftover in my testing...